PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1493,38 +1493,42 @@ intptr_t CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARA
 			if (LOWORD(wParam) == WA_ACTIVE || LOWORD(wParam) == WA_CLICKACTIVE)
 			{
 				Sci_CharacterRangeFull cr = (*_ppEditView)->getSelection();
-				intptr_t nbSelected = cr.cpMax - cr.cpMin;
+				LRESULT nbSelected = (*_ppEditView)->execute(SCI_COUNTCHARACTERS, cr.cpMin, cr.cpMax);
 
-				_options._isInSelection = isCheckedOrNot(IDC_IN_SELECTION_CHECK)?1:0;
-				int checkVal = _options._isInSelection?BST_CHECKED:BST_UNCHECKED;

```

Comment:
Restore the above lines please.

Reply:
What is not liked here?

Surely the addition of whitespace around `?` and `:` in (existing) line 1499 is not a problem...

That leaves the removal of `?1:0` from (existing) line 1498 as what is objected to.
But that is nonsensical as `isCheckedOrNot()` returns `bool` and `options._isInSelection` is a bool.
So there is no pure numeric type here and thus no need to involve numeric constants `1` and `0`.
I'm truly lost...

Reply:
The goal is to make minimum modification so the code review will focus on the essential stuff. Except the critical stake overflow or leak memory issue, it's suggested to keep the original code intact. 
[In short: The easier the code review is, the better the chance your pull request will get accepted.](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests)

Reply:
> it's suggested to keep the original code intact.

So, bad but working old code is allowed to stand, when there is an opportunity to make it better?
(Surely no one can argue that the modifications I suggested to these two lines don't make it better?)

No one would make a PR to only make such a change, but while someone is in there making changes to the general area, and theoretically testing the entire change well, IMO such changes make a great deal of sense.

Reply:
I'll do a refactoring after. In the meanwhile, let's focus on the essential part.

Reply:
If you will refactor after, can I propose a list of such changes, with rationale for each?  Then, much like typos, you can pick-and-choose from that list.

Reply:
@alankilborn 
Please do your list in an issue, then ping me. I will assign to myself for sure. Also there might be some points that we don't agree, so we can talk directly in the issue.